blocks/api/test/factory.js

```diff
@@ -21,6 +21,11 @@ describe( 'block factory', () => {
 		title: 'block title',
 	};
 
+	beforeAll( () => {
+		// Load all hooks that modify blocks
+		require( 'blocks/hooks' );
```

Comment:
Curious, would this be something we'd want to consider moving into general test setup? Seems easy to overlook when adding new tests.

Reply:
Yes, that's a valid question. I don't have an answer yet :)

My concern is that whenever we add this kind of code, it gives me an impression that we are doing integration tests rather than unit tests. Saying that, I'm wondering if creating a new integration tests configuration which loads all blocks and hooks on startup would be a good idea here. We need to come up with a wider strategy for all the different types of testing.